tests/bluetooth/tester/src/btp/btp_bap.h

```diff
@@ -20,6 +20,14 @@ struct btp_bap_discover_cmd {
 #define BTP_BAP_DISCOVERY_STATUS_SUCCESS	0x00
 #define BTP_BAP_DISCOVERY_STATUS_FAILED		0x01
 
+#define BTP_BAP_SEND	0x03
```

Comment:
Outside the scope of this PR, but I see that the events are prefixed with `BTP_BAP_EV` but the commands are just `BTP_BAP` instead of `BTP_BAP_CMD`. I assume this is a common naming theme for the bttester? 

@sjanc Would it make sense to prefix all commands with `BTP_XXX_CMD` similar to `BTP_XXX_EV` to make it clear that they represent commands?

Reply:
yeap, we could do that (just like we already do for corresponding structs)